apps/dashboard/app/models/posix_file.rb

```diff
@@ -3,7 +3,7 @@ class PosixFile
 
   attr_reader :path, :stat
 
-  delegate :basename, :descend, :parent, :join, :to_s, :read, :write, :mkdir, :directory?, :realpath, :pipe?, :readable?, to: :path
+  delegate :basename, :descend, :parent, :join, :to_s, :read, :write, :mkdir, :directory?, :realpath, :pipe?, :readable?, :file?, to: :path
```

Comment:
Delegation to `:pipe?` is probably not needed with this commit? I didn't find the symbol used elsewhere with a quick look.

Reply:
Yea it was for the other fix. I've removed it.